web3-actions/actions/trigger.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios'
+import { ActionFn, Context, Event, TransactionEvent } from '@tenderly/actions'
+
+const CROWDSALE_CONTRACT_MAINNET = '0x35Bce29F52f51f547998717CD598068Afa2B29B7'
+const CROWDSALE_CONTRACT_GOERLI = '0x46c3369dEce07176Ad7164906D3593AA4C126d35'
+const TOPIC0 =
+  '0xdcd726e11f8b5e160f00290f0fe3a1abb547474e53a8e7a8f49a85e7b1ca3199'
+
+const CROWDSALE_BIDS_QUERY_ID = '2709374'
```

Comment:
don't know about dune's capabilities but these seem like context variables to me. not secret but also not necessarily part of the code.